### DISCUSSION TITLE
GPT-4 API Access!

### DISCUSSION THREAD

**Discussion by edwardnewgate576** [mayooear/ai-pdf-chatbot-langchain#361] [General]
If anyone is looking for GPT-4 API Access then let me know. I am probably full access with ownership. 

P.S. - It's paid, not free.

**Comment by tonymynd** [comment]
Thanks, I am looking for GPT-4 Access (i don't know if I can afford till I see costs)

**Reply by edwardnewgate576** [reply]
> Thanks, I am looking for GPT-4 Access (i don't know if I can afford till I see costs)

Let me know where we can connect.

**Comment by rodneysdf** [comment]
@edwardnewgate576 Are you saying you can speed up getting API access to GPT-4? I'm still waiting for it...

**Reply by edwardnewgate576** [reply]
No,actually I have access and I am looking to sell it. If you are interested then let me know

**Comment by aim-soranut** [comment]
I am interested. How can we get in contact?

**Reply by edwardnewgate576** [reply]
> I am interested. How can we get in contact?

On what platform are you active? Telegram, whatsapp, discord?

**Reply by aim-soranut** [reply]
+12484047244 telegram
